src/Development/IDE/Core/Compile.hs

```diff
@@ -342,7 +333,7 @@ getModSummaryFromBuffer fp contents dflags parsed = do
     , ms_parsed_mod   = Nothing
     }
     where
-      (sourceType, derivedFile) =
+      (sourceType, _derivedFile) =
```

Comment:
This can be simplified further by dropping the `\newExt` stuff and using `takeExtension`. Any reason not to?

Reply:
Good point, thanks